README.rst

```diff
@@ -140,6 +163,8 @@ OpenSlides uses the following projects or parts of them:
 
 * `html5lib <https://github.com/html5lib/html5lib-python>`_, License: MIT
 
+* `channels <https://github.com/andrewgodwin/channels/>`_, License: MIT
```

Comment:
The project seems to be called only channels. So would keep it. Or do you have any source, where it is called "Django Channels"? I don't know if channels can be used for other APIs like flask.